client/src/modules/cohort/model/actions.js

```diff
@@ -219,6 +219,7 @@ export const deleteCohort = cohortId => dispatch => {
         NotificationType.ERROR,
         err.message || "Oops, we're sorry, deleting cohort failed..."
       );
+      throw err;
```

Comment:
should be throw new Error(err)  ?

Reply:
no because err is the Error throwing by server we throwing it forward...